tests/test_update.py

```diff
@@ -165,3 +191,114 @@ def test_get_pkg_version_none():
     )
 
     assert ver is None
+
+
+def test_get_pkg_version_komodo_dir_is_none():
+    ver = update.get_pkg_version(
+        {"python-version": "{}.{}".format(*sys.version_info)},
+        Path("/nonexisting/path/here"),
+        "pip",
+    )
+    assert ver is None
+
+
+def test_read_config_with_valid_file():
+    with patch(
+        "komodoenv.update.open", new_callable=mock_open, read_data=CONFIG_CONTENT
+    ):
+        config = update.read_config()
+        assert config["key1"] == "value1"
+        assert config["key2"] == "value2"
+        assert config["key3"] == "value3"
+        assert "malformed_line" not in config
+        assert config["komodo-root"] == "/prog/res/komodo"
+
+
+def test_rhel_version_suffix_with_distro_not_installed():
+    with patch.dict("sys.modules", {"distro": None}), patch(
+        "platform.release", return_value="3.10.0-1062.el7.x86_64"
+    ):
+        from komodoenv import update
+
+        importlib.reload(update)
+        assert update.rhel_version_suffix() == "-rhel7"
+    with patch.dict("sys.modules", {"distro": None}), patch(
+        "platform.release", return_value="4.18.0-147.el8.x86_64"
+    ):
+        from komodoenv import update
+
+        importlib.reload(update)
+        assert update.rhel_version_suffix() == "-rhel8"
+
```

Comment:
Could this be done using @pytest.mark.parametrize ?
Seems like the difference is just return_value and suffix for the two cases?